### PR TITLE
Feature/pathvariables

### PR DESCRIPTION
**Pull request by steverobprice**
Added variables that can be used in the watch path and copy to path. Gives users the ability to share config files and only have to update paths in one spot.

### CODE REVIEW

**Comment by steverobprice** [comment]
Resolves issue #7

**Comment by sbarski** [comment]
Looks good @steverobprice!!! Nice work :)

One thing - I had a look at actor.ask and it is doable. We can use it to substitute variables. All you need to do is use something like task.run and it should work.

E.g. var result = Task.Run(() => _substituteActor.Ask<string>(location.WatchFolder)).Result;

I just tried it before and it worked. So, I am thinking that we can refactor the substitute code in to an actor? What's your opinion on this?

**Comment by steverobprice** [comment]
Cool @sbarski I'll look to move my code into an actor. I think the more we can move into actors the better!

**Comment by steverobprice** [comment]
@sbarski I've now made that change to move the substitution code into its own actor. I've also added a few extension methods to help with calling the actor via Task.Run(). Let me know what you think.

File: src/watch.actor/ActorExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using Akka.Actor;
+using System;
+using System.Threading.Tasks;
+
+namespace watch.actor
+{
+    public static class ActorExtensions
+    {
```
Comment (sbarski) [inline src/watch.actor/ActorExtensions.cs:8]: This is nice! The funny thing is that if we were using an async env (e.g. asp.net async) then we could just talk to the actors directly without these extension methods. But alas!

**Pull request merged**